### PR TITLE
Added notices for preload page

### PR DESCRIPTION
**Pull request by donnchawp**

### CODE REVIEW

File: rest/class.wp-super-cache-rest-get-notices.php [hunk 1]
```diff
@@ -28,6 +29,40 @@ public function callback( $request ) {
...
+		if ( false == $super_cache_enabled ) {
+			$notices[ 'preload_disabled_supercache_off' ] = array(
+				'type' => 'warning',
+				'message' => __(
+					'Preloading is disabled as supercaching is disabled',
+					'wp-super-cache'
+				),
+			);
+		}
+		if ( false === defined( 'DISABLESUPERCACHEPRELOADING' ) ) {
```
Comment (donnapep) [inline rest/class.wp-super-cache-rest-get-notices.php]: @donnchawp Should this be checking for `true === defined( 'DISABLESUPERCACHEPRELOADING' )`?

File: rest/class.wp-super-cache-rest-get-notices.php [hunk 2]
```diff
@@ -28,6 +29,40 @@ public function callback( $request ) {
...
+		if ( false == $cache_enabled ) {
+			$notices[ 'preload_disabled_cache_off' ] = array(
+				'type' => 'warning',
+				'message' => __(
+					'Preloading is disabled as caching is disabled',
+					'wp-super-cache'
+				),
+			);
+		}
+		if ( false == $super_cache_enabled ) {
```
Comment (donnapep) [inline rest/class.wp-super-cache-rest-get-notices.php]: @donnchawp How do I enable supercache in order to test w/o this notice? This line is already in `wp-cache-config.php`:

`$super_cache_enabled = true;`

File: rest/class.wp-super-cache-rest-get-notices.php (hunk 1 above)
Comment (donnchawp) [inline rest/class.wp-super-cache-rest-get-notices.php]: You're right! Good catch!

File: rest/class.wp-super-cache-rest-get-notices.php (hunk 2 above)
Comment (donnchawp) [inline rest/class.wp-super-cache-rest-get-notices.php]: If you enable legacy mode caching that will disable super cache mode, and set `$super_cache_enabled = false;`

Comment (donnapep) [inline rest/class.wp-super-cache-rest-get-notices.php]: I can get it to show, but I can't get it to go away. 😄 Switching to PHP or mod_rewrite still returns the `preload_disabled_supercache_off` notice.

**Pull request merged**